embulk-core/src/main/java/org/embulk/spi/util/ReporterPlugins.java

```diff
@@ -0,0 +1,58 @@
+package org.embulk.spi.util;
+
+import com.google.common.collect.ImmutableMap;
+import com.google.common.collect.Maps;
+import org.embulk.config.ConfigSource;
+import org.embulk.config.TaskSource;
+import org.embulk.spi.Reporter;
+import org.embulk.spi.ReporterPlugin;
+
+import java.util.Map;
+
+public class ReporterPlugins
```

Comment:
Any specific reasons to have this `ReporterPlugins` utility class? I'm concerned:

1. If the methods in this `ReporterPlugins` are `public` in `org.embulk.spi`, they may look open for plugin developers. They're actually core's internal utility methods, though. If they're really just utility methods for internal use, they'd be `private` where they're used.
2. This is an all-`static` utility class in contrast to `Reporters` which is instantiatable for a set of `Reporter`s. It might be confusing.